.circleci/config.yml

```diff
@@ -127,6 +127,7 @@ jobs:
             build\distributions\besu\bin\besu.bat --version
 
   unitTests:
+    parallelism: 2
```

Comment:
This will just run all the tests on two boxes because there's nothing that's splitting the tests across them.  Se the `CLASSNAMES=$(circleci tests glob` stuff in the `acceptanceTests` task.

Reply:
yes of course. ok prob isn't worth spending time on now then.